src/query/pipeline/sources/src/processors/sources/input_formats/input_pipeline.rs

```diff
@@ -182,7 +182,7 @@ pub trait InputFormatPipe: Sized + Send + 'static {
                     break;
                 };
             }
-            tracing::info!("end copy splits feeder");
+            tracing::debug!("end copy splits feeder");
```

Comment:
Use `debug` instead of `tracing::debug`.

Reply:
Let's keep the style for input_formants.